ui/src/views/About.vue

```diff
@@ -208,39 +208,10 @@ export default {
     next();
   },
   methods: {
-    async getModuleInfo() {
+    getModuleInfo() {
       this.loading.moduleInfo = true;
-      const taskAction = "get-module-info";
-
-      // register to task completion
-      this.core.$root.$once(
-        taskAction + "-completed",
-        this.getModuleInfoCompleted
-      );
-
-      const res = await to(
-        this.createClusterTaskForApp({
-          action: taskAction,
-          data: {
-            id: "kickstart", // TODO
-          },
-          extra: {
-            title: this.$t("action." + taskAction),
-            isNotificationHidden: true,
-          },
-        })
-      );
-      const err = res[0];
-
-      if (err) {
-        console.error("error retrieving moodule info", err);
-        this.error.moduleInfo = this.getErrorMessage(err);
-        this.loading.moduleInfo = false;
-        return;
-      }
-    },
-    getModuleInfoCompleted(taskContext, taskResult) {
-      this.app = taskResult.output;
+      const metadata = require("../../public/metadata.json");
```

Comment:
As the file is included at build time: is it necessary to place it under public/? Can we use a directory that is not served via  HTTP?

Reply:
WHy?

Reply:
Accessing that .json file at build-time may lead to a different result from accessing it at run-time, via HTTP